tap_jira/http.py

```diff
@@ -141,13 +144,26 @@ def check_status(response):
             response.status_code, {}).get("raise_exception", JiraError)
         raise exc(message, response) from None
 
+def get_request_timeout(config):
+    # Get `request_timeout` value from config
+    config_request_timeout = config.get('request_timeout')
+
+    # if config request_timeout is other than 0, "0", or "" then use request_timeout
+    if config_request_timeout and float(config_request_timeout):
+        request_timeout = float(config_request_timeout)
+    else:
+        # If value is 0, "0", "", or not passed then it set default to 300 seconds
+        request_timeout = REQUEST_TIMEOUT
```

Comment:
Is there a reason we use `float()` but the default value is an integer?

Reply:
When we tested the code with integer datatype and passed the value of request_timeout as "120.0" in the config file, then the below error message is thrown.
CRITICAL invalid literal for int() with base 10: '120.0'. 
Also, we were not sure about the data type returned from the UI. So, to handle all the scenarios we used float datatype